pkgs/build-support/ocaml/dune.nix

```diff
@@ -25,12 +25,12 @@ stdenv.mkDerivation ({
     runHook postInstall
   '';
 
-  meta.platform = ocaml.meta.platform;
-
 } // args // {
 
   name = "ocaml${ocaml.version}-${pname}-${version}";
 
   buildInputs = [ ocaml dune findlib ] ++ buildInputs;
 
+  meta = args.meta // { platforms = args.meta.platforms or ocaml.meta.platforms; };
```

Comment:
Don't you need to add `, meta ? { }` to the argument list on line 3 to make sure that at least `args.meta` exists?

Reply:
Well spotted, thanks.